.buildkite/autogenerate_pipeline.py

```diff
@@ -78,8 +83,8 @@
 # More details here: https://github.com/rust-vmm/community/issues/137
 DEFAULT_AGENT_TAG_HYPERVISOR = os.getenv("DEFAULT_AGENT_TAG_HYPERVISOR", "kvm")
 
-PARENT_DIR = pathlib.Path(__file__).parent.resolve()
-
+BUILDKITE_PATH = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I can't put a comment on line 55, but `pathlib` is not used anymore after this change.

BTW why are we switching from pathlib to os.path? (I'm not a python expert at all)
IMHO we should do that change in a separate commit, and explain why.

Reply:
The `os.path` way is derived from `test_coverage.py`, I'm switching it back to `pathlib`